opencti-platform/opencti-graphql/src/manager/taskManager.js

```diff
@@ -265,16 +266,14 @@ const sendResultToQueue = async (context, user, task, objects, opts = {}) => {
     no_split: opts.forceNoSplit ?? false
   });
 };
-
-const buildBundleElement = (element, actionType, operations) => {
+const buildBaseBundleElement = (element, standard_id) => {
```

Comment:
the two usage of this method add additional information in extensions[STIX_EXT_OCTI], it could directly added here via a second args that will receive the additional information to be added for this ext